interfaces/cython/cantera/test/test_composite.py

```diff
@@ -133,6 +124,13 @@ def setUpClass(cls):
         utilities.CanteraTest.setUpClass()
         cls.gas = ct.Solution('h2o2.yaml', transport_model=None)
 
+    def tearDown(self):
+        temp_files = ["solutionarray.csv", "solutionarray.h5"]
```

Comment:
I think you can simplify this as:

```python
for p in self.test_work_path.glob("solutionarray.*"):
    p.unlink()
```

assuming that `self.test_work_path` is a `PurePath` instance.

Reply:
I actually just changed this one to deleting the old files at the last possible moment (to make sure that files exist after running a single test).